common.yml

```diff
@@ -1319,6 +1319,13 @@ tasks:
       vars:
         test_path: "test/legacy42"
 
+- name: mod-tidy
+  commands:
+    - func: "fetch source"
+    - func: "run make target"
+      vars:
+        target: sa:modtidy
```

Comment:
Do you think it's worth renaming "run make target" to something else like "run target", since it's now the entry point for building, testing, and static analysis?

Reply:
I don't think so; I don't think there's anything about having a static analysis task that makes it more or less of a "make" target.